### PR TITLE
Correct pull request template per your request

### PR DESCRIPTION
**Pull request by keitheous**
:v:

/cc @zendesk/vegemite

### Description
Remove the construction emoji and change the wording in Risks. 

### Risks
* [low] - no risks as this only affects PRs on Github.

### CODE REVIEW

**Comment by keitheous** [comment]
yay, @sostopher !

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -13,8 +13,8 @@ Describe the original problem and the changes made on this PR.
 * JIRA: https://zendesk.atlassian.net/browse/AF-XXX
 
 ### Risks
-* [HIGH | medium | low] Does it work across browsers (including IE!)?
-* [HIGH | medium | low] Does it work in the different products (Support, Chat, Connect)?
```
Comment (token-cjg) [inline .github/pull_request_template.md]: Are we planning on having apps in Connect at some point?

Comment (keitheous) [reply .github/pull_request_template.md]: Hi @token-cjg ,  You're right. Thanks for noticing. I realised this and changed `Connect` to `Guide` in the latest push.

Comment (Mehul-Porwal) [reply .github/pull_request_template.md]: Guide is also not confirmed yet. So maybe leave it to support and chat for now.

**Pull request merged**